<div class="article">
  <div class="article_head-line">
    <h1>Attributions</h1>
  </div>
  <h2>Outside of the team roster</h2>
  <div class="article_picture article_picture-right">
    <img src="http://2018.igem.org/wiki/images/1/13/T--Goettingen--attributions-genomics.png"
         alt="Göttingen Genomics Laboratory">
    <p>Göttingen Genomics Laboratory</p>
  </div>
  <h3>RNA sequencing</h3>
  <p>For acquiring RNA sequencing data, we asked <strong>Professor Rolf Daniel</strong> from Göttingen Genomics Laboratory (G2L) to support our project. He provided us a lot of RNA sequencing data that enabled us to overthink our detection method with reporter genes.</p>
  <h3 class="article_clear-right">
    ITC measurements</h3>
  <p>For isothermal titration calorimetry (ITC) measurements and structure elucidation of our purified proteins AroA from <em>Escherichia&nbsp;coli</em> and AroE from <em>Bacillus&nbsp;subtilis</em> we sent our samples to the Bange lab in Marburg. <strong>Gert Bange</strong> is also supervising an iGEM team this year and will hopefully provide us with data? TO DO
  </p>
  <div class="article_picture article_picture-right">
    <img src="http://2018.igem.org/wiki/images/c/cb/T--Goettingen--attributions-seqlab.png"
         alt="Seqlab Göttingen">
    <p>Seqlab Göttingen</p>
  </div>
  <h3>DNA sequencing</h3>
  <p>To acquire DNA sequences, we sent our samples as described in the notebook section to <em>Microsynth Seqlab</em>, based in Göttingen.
  </p>
  <h3 class="article_clear-right">
    Detection of glyphosate </h3>
  <p>For the detection of glyphosate, we sent our samples to the <strong>Ischebeck lab</strong> in Göttingen, which provided us with the possibility to use their gas chromatography and mass spectrometry devices for determining the concentration of glyphosate in samples. TO DO
  </p>
  <h3>Genes</h3>
  <p>We ordered some genes, called g-Blocks, from <em>Integrated DNA Technologies</em>, which was free, since they sponsor the iGEM competition.
  </p>
  <h3>Contributions from the lab members</h3>
  <p>We want to thank <strong>Sabine Lentes</strong> for enduring eight people in the lab with many questions. She helped us in every situation and was a great aid.<br /> Furthermore, we want to thank <strong>Johannes Gibhardt</strong> for his help with any technical devices that we used. Especially he is an expert for the platereader.<br /> <strong>Christina Herzberg</strong> supported us with strains and plasmids, knowledge about technical methods, and answered many questions.<br /> <strong>Cedric Blötz</strong> helped us with his experience of being a former iGEM participant and for helpful discussions regarding our project.<br />
    <strong>Anika Klewing</strong>helped us with buffers, kits and chemicals that we needed every day.<br /> Last but not least we want to thank all members from the Stülke lab for their support and endorsement in the time we used their lab and blocked technical devices with our experiments.
  </p>
  <h2>Work from team roster</h2>
  <p>We did all of the lab work by ourselfs as described in the notebook section. The experiments were developed in teamwork with our supervisors.</p>
  <p>The over all design was drafted by ourselves. Also, the wiki and website were constructed by us.
  </p>
  <h3>Supervisors</h3>
  <p>The whole project is based on the ideas from <strong>Dr. Fabian Commichau</strong> that came up to his mind on a train ride to Brussels. We want to thank him for his infectious enthusiasm towards microbiology and great help with each question!</p>
  <p>We want to thank also <strong>Prof. Dr. Jörg Stülke</strong> for kindly providing us with a laboratory, which is obviously necessary for such a project. Furthermore, he greatly supported us with the funding and his experience in leading former iGEM teams.</p>
  <h3>Construction of strains</h3>
  <p>We want to thank Dr. Fabian Commichau and Sabine Lentes for help with the construction of the deletion strains that were needed for our work.
  </p>
</div>
